www-nginx/ngx-lua-module/ngx-lua-module-0.10.27.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="lua-nginx-module"
+inherit lua-utils nginx-module
```

Comment:
Again, lua-single here (please also check all my comments to see if they need to be applied throughout)